packages/eslint-plugin/tests/rules/prefer-nullish-coalescing.test.ts

```diff
@@ -20,6 +20,7 @@ const ruleTester = new RuleTester({
 
 const types = ['string', 'number', 'boolean', 'object'];
 const nullishTypes = ['null', 'undefined', 'null | undefined'];
+const ignorablePrimitiveTypes = ['string', 'number', 'boolean'] as const;
```

Comment:
[Nitpick] doesn't look like this is needed?

```suggestion
const ignorablePrimitiveTypes = ['string', 'number', 'boolean'];
```